include/rocksdb/advanced_options.h

```diff
@@ -555,7 +555,7 @@ struct AdvancedColumnFamilyOptions {
 
   // NOT SUPPORTED ANYMORE
   // This does not do anything anymore.
-  int max_mem_compaction_level;
+  // int max_mem_compaction_level;
```

Comment:
Can you instead initialize this option to 0? It is leave here to avoid users hitting compile error.

Reply:
@yiwu-arbug I have commented out all the references to max_mem_compaction_level variable from struct AdvancedColumnFamilyOptions. Would we still want to use max_mem_compaction_level variable ?

Reply:
Let's keep max_mem_compaction_level for now.

Reply:
@yiwu-arbug made the changes. There are 2 commits on this PR, but pick 37f163c only. Let me know if you want me to open another PR if things are messy.